internal/repositories/vehicles.go

```diff
@@ -191,15 +190,18 @@ func (v *Repository) GetVehicles(ctx context.Context, first *int, after *string,
 			return nil, err
 		}
 	} else {
-		// TODO(elffjs): Ugly.
-		queryMods = []qm.QueryMod{}
```

Comment:
I think this is here for a reason? Hm, maybe not

https://go.dev/play/p/76sz0-OLykg